### ISSUE TITLE
Improve sector weighting in InfluenceMap Score (to tackle distortion coming from O&G)

### ISSUE THREAD

**Issue by 2diiKlaus** [2DegreesInvesting/PACTA_analysis#16]
**Is your feature request related to a problem? Please describe.**
Currently we weight the sector level results by the value (USD), i.e. the financial exposure, to the sector. As the average market (at least in equity) is much more exposed to the oil&gas sector, the results of this sector are driving the rating. As we weight the importance of each sector using its transition importance already, this second weighting will overrate the oil&gas importance. 

We need to improve/remove this additional weighting

This needs to be done after #15 is included

**Describe the solution you'd like**
There are 3 alternative approaches how we can tackle it:

1. Use the deviation from the market to include the financial exposure importance of the sector (i.e. if the market is 15% exposed to O&G and the fund 16% the weighting would be 16/15)
2. Use the absolute production of the entire client base compared to the market (i.e. the sum of the entire production of all counter-parties divided by the sum of all production in the sector as a whole using the corporate economy, i.e. all ALD we have)
3. Not use any financial weight at all (this can lead to distortion as well as having 0.0001% to the power sector and only have a pure-play renewable could make you look good  despite any real contribution/exposure to the power sector)

We want to explore 1 and 2 and understand the differences.

**Describe alternatives you've considered**
see above number 3 as well as the current version

**Additional context**
Slack chat vincent&klaus 06.04.2020 - 7 AM (search for "Looking at the fund above made me realize that weighting by the sector value USD introduces some counter-intuitive results.")

**Can you provide something like this?**
Not now

```
#' A title line, for example: Sum two numbers
#' 
#' One description paragraph. For example: This function sums two numbers. It
#' does nothing really useful other than showing how to document a function.
#'
#' Optionally, any number of paragraphs giving details. We follow the tidyverse
#' style guide: <https://style.tidyverse.org/>.
#' 
#' @param x Definition of the `x` argument. For example: A number.
#' @param y Definition of the `y` argument. For example: A number.
#'
#' @return What do you expect this function to return? For example: A number.
#' @export
#'
#' @examples
#' sum_two_numbers(1, 1)
sum_two_numbers <- function(x, y) {
  stopifnot(is.numeric(x), is.numeric(y))
  
  x + y
}
```
